tests/model_selection/test_hyperband.py

```diff
@@ -29,6 +29,7 @@
 pytestmark = pytest.mark.skipif(not DISTRIBUTED_2_5_0, reason="hangs")
 
 
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
These should be updated too.

Reply:
Thanks a lot for the review. Will add that.

Reply:
Removed with commit https://github.com/dask/dask-ml/pull/862/commits/094c31adc3352c50966f86d501739b1ea50a9c9e